app/gui/src/project-view/components/GraphEditor/widgets/WidgetSelection/SelectionSubmenu.vue

```diff
@@ -71,7 +71,9 @@ function nestedEntryToSubmenu(entry: SubmenuEntry<T>, target: HTMLElement): Subm
 
 function onClick(entry: T, keepOpen: boolean, htmlElement: HTMLElement) {
   if (isSubmenuEntry(entry) && entry.isNested) {
-    submenu.value = nestedEntryToSubmenu(entry, htmlElement)
+    // Change submenu in two steps to trigger the size transition.
+    submenu.value = null
+    nextTick(() => (submenu.value = nestedEntryToSubmenu(entry, htmlElement)))
```

Comment:
Is there a better way to trigger SizeTransition?

Reply:
Changing the `:key` should work